src/main/java/org/folio/rmapi/PackageServiceImpl.java

```diff
@@ -25,15 +27,31 @@
 
   private ProvidersServiceImpl providerService;
   private TitlesHoldingsIQService titlesService;
+  private VertxCache<PackageCacheKey, PackageByIdData> packageCache;
+  private Configuration configuration;
+  private String tenantId;
 
-  public PackageServiceImpl(Configuration config, Vertx vertx, ProvidersServiceImpl providerService, TitlesHoldingsIQService titlesService) {
+  public PackageServiceImpl(Configuration config, Vertx vertx, String tenantId, ProvidersServiceImpl providerService,
+                            TitlesHoldingsIQService titlesService, VertxCache<PackageCacheKey, PackageByIdData> packageCache) {
     super(config, vertx);
     this.providerService = providerService;
     this.titlesService = titlesService;
+    this.packageCache = packageCache;
+    this.tenantId = tenantId;
+    this.configuration = config;
   }
 
   public CompletableFuture<PackageResult> retrievePackage(PackageId packageId, List<String> includedObjects) {
-    CompletableFuture<PackageByIdData> packageFuture = retrievePackage(packageId);
+    return retrievePackage(packageId, includedObjects, false);
```

Comment:
I am just wondering can we make the default lookup of a package in cache? If its not present in cache, we anyways reach out to RM API. What do you think?

Reply:
We can use cache by default, but I wanted to use cache only in case when we are getting tagged packages, 
then all other features will still have live data, and cache will be smaller, because it will only contain tagged packages

Reply:
ok, makes sense 👍